scripting/get5.sp

```diff
@@ -534,6 +543,18 @@ public Action Event_PlayerConnectFull(Event event, const char[] name, bool dontB
   }
 }
 
+public Action Event_PlayerDisconnect(Event event, const char[] name, bool dontBroadcast) {
+  int client = GetClientOfUserId(event.GetInt("userid"));
+  g_ClientsConnected--;
+  EventLogger_PlayerDisconnect(client);
+
+  if (g_ClientsConnected <= 0) {
```

Comment:
Now that I look at this again, I'd worry about this firing during map changes. Have you tested to make sure that doesn't happen? I guess you could also check to make sure g_MapChangePending if that's an issue.